common/src/test/resources/ssl/generate_certs.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A simple utility to (re-)generate the files within resources/ssl
+# These generated certificates are used for a variety of test scenarios for SSL.
+# The utility ends up generating two certificates - which are saved into two different keystores
+# The certificates generated are signed by two different CA cert's (also generated here).
+# There are two truststores generated - the first truststore has both CA certs as part of it
+# Hence this trust can be used to validate both client certificates.
+# The second trust store has NO CA certs in it - and so when used will fail both the certificates.
+
+function gen_certs() {
+
+  openssl genrsa -out ca.key 2048
+  openssl req -x509 -new -days 9000 -key ca.key -out ca.crt   -subj "/C=US/ST=YourState/L=YourCity/O=YourOrganization/CN=MyCACert"
+  openssl genrsa -out server.key 2048
+  openssl req -new -key server.key -out server.csr   -subj "/C=US/ST=YourState/L=YourCity/O=YourOrganization/CN=MyServer"
```

Comment:
nit: some places use more than one space, if you don't mind, please fix the format
```suggestion
  openssl req -new -key server.key -out server.csr -subj "/C=US/ST=YourState/L=YourCity/O=YourOrganization/CN=MyServer"
```